.github/workflows/publish-release.yml

```
@@ -133,6 +172,7 @@ jobs:
     needs:
       - prepare-release-notification
     if: ${{ needs.prepare-release-notification.outputs.available == 'true' }}
```

actionlint (1.7.12)

[error] if: condition "${{ needs.prepare-release-notification.outputs.available == 'true' }} - publish-helm-chart" is always evaluated to true because extra characters are around ${{ }}

(if-cond)